.github/workflows/build-and-test-windows.yml

```diff
@@ -0,0 +1,149 @@
+
+name: "MSYS2 Build & Test"
+
+on:
+  # allow direct trigger
+  workflow_dispatch:
+  push:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+env:
+  COMMON_CMAKE_FLAGS: >
+    -DSLEEF_SHOW_CONFIG=1
+    -DSLEEF_BUILD_GNUABI_LIBS=ON
+    -DSLEEF_BUILD_DFT=ON
+    -DSLEEF_BUILD_QUAD=ON
+    -DSLEEF_BUILD_SCALAR_LIB=ON
+    -DSLEEF_BUILD_STATIC_TEST_BINS=ON
+    -DSLEEF_ENFORCE_TESTER=ON
+    -DSLEEF_ENFORCE_TESTER3=ON
+
+jobs:
+  build-msys2:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sys:
+          - clang64
+          #- mingw64
+          #- ucrt64
```

Comment:
Could we have a quick comment saying why these are commented out?

Reply:
Do you have any more details on that, maybe a link? Is the compiler broken or is it SLEEF when compiled in a gcc envs?